content/applications/finance/payment_acquirers/adyen.rst

```diff
@@ -97,7 +105,4 @@ where payments will be made (the URLs of the servers hosting your Odoo instances
 
 .. image:: adyen/adyen_allowed_origins.png
    :align: center
-   :alt: Allows payments originated from a specific domain.
-
-.. seealso::
-   - :doc:`../payment_acquirers`
+   :alt: Allows payments originated from a specific domain.
```

Comment:
As for API and Client Keys, this also slightly changed as it's now in the `Client Settings` part, which is new. There are some small visual changes too here ^^